websubhub-examples/kafka-hub/consolidator/modules/config/configurations.bal

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import consolidatorService.util;
+
+# IP and Port of the Kafka bootstrap node
+public configurable string KAFKA_BOOTSTRAP_NODE = "localhost:9092";
+
+# Kafka topic which will get notified for websub topic registration/deregistration
+# All the hubs must be pointed to the same Kafka topic to notify websub topic registration/deregistration
+public configurable string REGISTERED_TOPICS_TOPIC = "registered-topics";
+
+# Kafka topic which stores consolidated websub topics for the hub
+public configurable string CONSOLIDATED_TOPICS_TOPIC = "consolidated-topics";
+
+# Kafka topic which will get notified for websub subscription/unsubscription
+# All the hubs must be pointed to the same Kafka topic to notify websub subscription/unsubscription
+public configurable string SUBSCRIBERS_TOPIC = "registered-subscribers";
```

Comment:
Is this comment correct ?

Reply:
Yes, this is the `topic` which every hub notifies for subscription/unsubscription